packages/access/src/ownable/ownable.cairo

```diff
@@ -311,12 +317,7 @@ pub mod OwnableComponent {
         fn _propose_owner(ref self: ComponentState<TContractState>, new_owner: ContractAddress) {
             let previous_owner = self.Ownable_owner.read();
```

Comment:
```suggestion
            assert(!new_owner.is_zero(), Errors::ZERO_ADDRESS_OWNER);

            let previous_owner = self.Ownable_owner.read();
```

Zero owner shouldn't be proposed, since there is a renounce mechanism for that, and the zero address can never accept after being proposed.

Reply:
We need to add tests for this and Requirements comment must be updated.